tests/Main.hs

```diff
@@ -367,7 +367,7 @@ runClashTest = defaultMain $ clashTestRoot
           --
```

Comment:
This comment also needs to go

Reply:
Wow I thought CI would take a while, it was done in under 32 minutes?

Reply:
ah flip, you're right.

Reply:
> 12 jobs for
[fix-2325-2334](https://gitlab.com/clash-lang/clash-compiler/-/commits/fix-2325-2334)
in 18 minutes and 50 seconds

GitLab :-). GHA usually takes longer, but it was feeling generous today!